tests/llmcompressor/modeling/test_calib_gemma4.py

```
@@ -0,0 +1,196 @@
+        moe_layer = None
+        for _, module in model.named_modules():
+            if isinstance(module, SequentialGemma4TextExperts):
+                moe_layer = module
+                break
+
+        assert moe_layer is not None
+
+        num_experts = moe_layer.num_experts
+        expert_triggered = [False for _ in range(num_experts)]
+
+        def hook_fn(i, module, input, output):
```

Ruff (0.15.9)

[error] Function argument `input` is shadowing a Python builtin

(A002)